### PR TITLE
Operationalize harprofiler.

### PR DESCRIPTION
**Pull request by benpatterson**
@cgoldberg @clytwynec 

Also if one of you has the time: @feanil @fredsmith

### CODE REVIEW

File: playbooks/roles/harprofiler/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  sudo_user: "{{ harprofiler_user }}"
+
+- name: set bashrc for harprofiler user
+  template: >
+    src=bashrc.j2 dest="{{ harprofiler_dir }}/.bashrc" owner="{{ harprofiler_user }}"
+    mode=0755
+
+- name: install requirements
+  pip: >
+    requirements="{{ harprofiler_dir }}/requirements.txt" virtualenv="{{ harprofiler_dir }}/venv"
```
Comment (feanil) [inline playbooks/roles/harprofiler/tasks/main.yml]: Nitpick: The pattern we've been using for applications here is '< app_dir >/venvs/<venv_name>', the idea being that we could have multiple Venvs for a particular application.

**Comment by feanil** [comment]
Other than nitpick, looks good from what I understand.

**Comment by fredsmith** [comment]
:+1: LGTM

File: playbooks/roles/harprofiler/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+#
+# edX Configuration
+#
+# code style: https://github.com/edx/configuration/wiki/Ansible-Coding-Conventions
+
+harprofiler_role_name: harprofiler
+
+harprofiler_user: "harprofiler"
+harprofiler_github_url: https://github.com/cgoldberg/harprofiler
```
Comment (clytwynec) [inline playbooks/roles/harprofiler/defaults/main.yml:10]: Just curious...  Are we still planning on moving/forking this to a repo under the edx org?

Comment (benpatterson) [inline playbooks/roles/harprofiler/defaults/main.yml:10]: Yep that's the plan AFAIK

File: playbooks/roles/harprofiler/tasks/main.yml (hunk 1 above)
Comment (benpatterson) [inline playbooks/roles/harprofiler/tasks/main.yml]: Done.

**Pull request merged**